app/resources/status.html.template

```diff
@@ -0,0 +1,83 @@
+{# Copyright 2010 Google Inc.  Licensed under the Apache License, Version   #}
```

Comment:
I'm trying to think what's the good name for this file. Since PFIF spec specifically calls out the concept of Person and Note, I think we should be using note as part of the file name. What about renaming this file to add_note.html.template and renaming the existing add_note.html.template to add_note_base.html.template? I believe the latter is now used only by included in other files?